cli/src/cli_util.rs

```diff
@@ -1942,9 +2015,10 @@ pub fn check_stale_working_copy(
         // The working copy isn't stale, and no need to reload the repo.
         Ok(WorkingCopyFreshness::Fresh)
     } else {
-        let wc_operation_data = repo
-            .op_store()
-            .read_operation(locked_wc.old_operation_id())?;
+        let wc_operation_data = match repo.op_store().read_operation(locked_wc.old_operation_id()) {
+            Ok(o) => o,
+            Err(_) => return Ok(WorkingCopyFreshness::MissingOperation),
```

Comment:
nit: better to check if the error is `ObjectNotFound`

Reply:
OK.